### PR TITLE
Initial machine and register information

### PR DESCRIPTION
**Pull request by pmatos**

### CODE REVIEW

File: s/rv64.ss [hunk 1]
```diff
@@ -0,0 +1,121 @@
+;;; rv64.ss
+;;; Copyright 2020 Paulo Matos
```
Comment (jessealama) [inline s/rv64.ss]: minor point: would you like to include your email address here, too? Or is it enough to roll with your name only?

Comment (jessealama) [reply s/rv64.ss]: What's the definition of done for this PR? Can I make .boot files for RISCV with these?

**Comment by pmatos** [comment]
@jessealama do you have any questions about this?

**Review by jessealama — APPROVED** [review]

File: s/rv64.ss (hunk 1 above)
Comment (pmatos) [reply s/rv64.ss]: I never know what to do about copyrights and it looks stupid here but other files don't include contact information. In WebKit it's the same, however this code might one day be included in racket repo and they might need to contact us, so lets add our email to all the files we touch. If one day we have to drop it, I don't really care that much.

Comment (pmatos) [reply s/rv64.ss]: Boot files can only be created after the whole ISA is defined, this is a tiny stepping stone towards that.

**Comment by pmatos** [comment]
Squashing and merging.

**Pull request merged**